src/DependencyInjection/Factory/BlockAdminFactory.php

```diff
@@ -70,10 +71,10 @@ public function create(array $config, ContainerBuilder $container, XmlFileLoader
 
         $bundles = $container->getParameter('kernel.bundles');
 
-        if (true === $config['enable_menu']
-            || ('auto' === $config['enable_menu'] && isset($bundles['CmfMenuBundle']))
-        ) {
+        if ($config['enable_menu'] && array_key_exists('CmfMenuBundle', $bundles)) {
             $loader->load('block-menu.xml');
+        } elseif (true === $config['enable_menu'] && !array_key_exists('CmfMenuBundle', $bundles)) {
+            throw new InvalidConfigurationException('To use the menu block, you need the symfony-cmf/menu-bundle in your project.');
         }
```

Comment:
the point is that we load on `true` or `"auto"` but only throw an exception on `true` but not on auto.